spec/amp-story-player.md

```diff
@@ -123,19 +123,22 @@ URL pointing to the story.
 
 ## Programmatic Control
 
-Call the player's various methods to programmatically control the player. These methods are exposed on the HTML element, `const playerEl = document.querySelector('amp-story-player')` and on instances of the global class variable, `const player = new AmpStoryPlayer(window, playerEl)`.
+Call the player's various methods to programmatically control the player. These methods are exposed on the HTML element, `const playerEl = document.querySelector('amp-story-player')`.
 
 ### Methods
 
 #### load
 
-Will initialize the player manually. This can be useful when the player is dynamically.
+Will initialize the player manually. This can be useful when creating the player dynamically.
 
-Note that the element must be connected to the DOM before calling `load()`.
+Note that the amp-story-player JS will automatically do this when the player is already in the HTML markup, so only do this when you really need to.
+
+Also note that the element must be connected to the DOM before calling `load()`.
 
 ```javascript
-const playerEl = document.body.querySelector('amp-story-player');
+const playerEl = document.createElement('amp-story-player');
 const player = new AmpStoryPlayer(window, playerEl);
+document.body.appendChild(playerEl);
 player.load();
```

Comment:
To be consistent and use the DOM element in both cases. In your example `playerEl === player` so it's weird to declare it twice

```suggestion
const playerEl = document.createElement('amp-story-player');
new AmpStoryPlayer(window, playerEl);
document.body.appendChild(playerEl);
playerEl.load();
```